Dropdown/package.json

```diff
@@ -35,21 +35,27 @@
     "babel-preset-es2015": "6.6.0",
     "babel-preset-react": "6.5.0",
     "cpy-cli": "^1.0.1",
+    "enzyme": "^3.1.1",
+    "enzyme-adapter-react-15.4": "^1.0.5",
+    "enzyme-to-json": "^3.2.2",
     "eslint": "2.10.2",
     "eslint-config-dnn": "^0.0.7",
     "eslint-import-resolver-node": "^0.3.1",
     "eslint-plugin-babel": "^4.1.2",
     "eslint-plugin-import": "^2.8.0",
-    "npm-run-all": "^4.1.1"
+    "eslint-plugin-jest": "^21.2.0",
+    "jest": "^21.2.1",
+    "npm-run-all": "^4.1.2",
+    "react-addons-test-utils": "^15.6.2"
   },
   "dependencies": {
     "dnn-global-styles": "0.0.5",
     "dnn-svg-icons": "0.1.20",
     "lodash": "4.17.4",
-    "react": "^15.3.2",
+    "react": "^15.6.2",
     "react-collapse": "~4.0.3",
-    "react-custom-scrollbars": "~4.1.2",
-    "react-dom": "^15.3.2",
+    "react-custom-scrollbars": "^4.2.1",
+    "react-dom": "^15.6.2",
     "react-motion": "^0.4.4",
     "scroll": "^2.0.0"
   }
```

Comment:
Some dependencies were missing and prepublishOnly script was failing. Now it's ok.